templates/support/livepatch.html

```diff
@@ -0,0 +1,160 @@
+{% extends "support/base_support.html" %}
+
+{% block title %}Canonical Livepatch Service{% endblock %}
+{% block meta_description %}Automatic updates for Ubuntu 14.04, 16.04, and 18.04 LTS, reducing downtime whilst keeping on top of compliance and security.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1ObUSvkDcWOe01pmhh_Aryqxs87D8X2khbJbn3RgqZE0/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-6">
+      <h1 style="font-weight: 100;">Canonical Livepatch Service</h1>
+      <h2 class="p-heading--four">Apply critical kernel patches without rebooting.</h2>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Fixes are applied automatically, without restarting your system</li>
+        <li class="p-list__item is-ticked">Reduces downtime, keeping your Ubuntu LTS systems secure and compliant</li>
+        <li class="p-list__item is-ticked">Included as part of all Ubuntu Advantage support packages</li>
+      </ul>
+    </div>
+    <div class="col-6">
+      <div class="u-embedded-media">
+        <iframe class="u-embedded-media__element" width="772" height="426" src="https://player.vimeo.com/video/225445183" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <hr style="margin: 5rem 0;" />
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-10">
+      <h2>Subscribe to Livepatch</h2>
+      <p class="u-no-max-width">
```

Comment:
I am not keen on the amount of bespoke styling there is for this section. Could you build it using the grid, please.

For example:
``` html
<div class="row">
  <div class="col-4">
    <p>
      <a href="https://auth.livepatch.canonical.com/" class="p-button--positive row"><span class="p-link--external">Free for up to 3 machines</span></a>
    </p>
  </div>
  <div class="col-8">
    <p>All you need is an Ubuntu One account.</p>
  </div>
</div>
```